### PR TITLE
Programmieruebung02

### PR DESCRIPTION
**Pull request by AaronHtlDornbirn**

### CODE REVIEW

File: SingleLinkedListTest/UnitTest1.cs [hunk 1]
```diff
@@ -3,19 +3,59 @@
...
 {
+
     public class Tests
     {
         [SetUp]
-        public void Setup() {}
+        public void Setup() { }
 
         [Test]
-        public void TestLinkedList_AddItems_ReturnsCorrectList()
```
Comment (martinernstmueller) [inline SingleLinkedListTest/UnitTest1.cs:12]: Warum hast du diesen Test nicht drinnen gelassen und einen neue für die switchnode-Methode geschrieben?

File: SingleLinkedListTest/UnitTest1.cs [hunk 2]
```diff
@@ -3,19 +3,59 @@
...
+        public void SwitchNodes_values_switched()
```
Comment (martinernstmueller) [inline SingleLinkedListTest/UnitTest1.cs:13]: Bessere Benennung z.B. SwitchNodes_NodesExist_NodesAreSwitched

Comment (martinernstmueller) [reply SingleLinkedListTest/UnitTest1.cs:13]: Der Test schlägt fehl, weil die Nodes nicht geswitched werden

**Review by martinernstmueller — COMMENTED** [review]
Funktionalität:
Compiliert nicht (hab in meinem korrektur-Branch den Rückgabetype auf void geändert), aber auch dann switched die Methode die übergebenen Nodes in der Liste nicht -> 0.5/3
Unit-Tests: Benennung nicht gut (siehe Kommentar), Klasse und Projekt haben auch noch keine aussagekräftigen Namen. Es sind auch zu wenige tests... es sollte auch "edge-cases" getestet werden -> 1/2
Clean Code: Benennung der Methoden und Variablen ist ok, Einrückungen und Zeilenumbrüche auch -> 2/2
